src/music/Drums.ts

```diff
@@ -0,0 +1,58 @@
+import { AdsrOscillatorInstrument, SampleInstrument } from "./Instruments.js";
+import { Samples, SampleNames } from "./Samples.js";
+
+export enum Drums {
```

Comment:
`DrumType` may be a more accurate name, but if you think this works better, that's fine too

Reply:
`Drums` feels more in line with the rest of the codebase, though I may be wrong there.